test/IntegrationTests.t.sol

```diff
@@ -292,17 +305,17 @@ abstract contract ExecuteOnceSpellTests is IntegrationTestsBase {
 
     function test_cannotCallTwice() external {
         vm.prank(SPARK_PROXY);
-        if (isPauseSpell) aclManager.addEmergencyAdmin(address(freezer));
-        else aclManager.addRiskAdmin(address(freezer));
+        if (isPauseSpell) aclManager.addEmergencyAdmin(address(freezerMom));
+        else aclManager.addRiskAdmin(address(freezerMom));
```

Comment:
Can remove these lines now because of the setUp

Reply:
done